scipy/ndimage/tests/test_ni_converters.py

```diff
@@ -0,0 +1,204 @@
+from __future__ import division, print_function, absolute_import
+
+import gc
+import itertools
+
+import numpy as np
+from numpy.testing import (assert_, assert_array_equal, assert_raises_regex,
+                           run_module_suite)
```

Comment:
Maybe don't add run_module_suite at all, so I don't have to remove it in gh-7572